lib/posix/CMakeLists.txt

```diff
@@ -2,3 +2,70 @@
 
 add_subdirectory(options)
 add_subdirectory(shell)
+
```

Comment:
looks like a rebase error to me, just create a new branch of the latest `main` (make sure you synced the upstream), and work from there